website/templates/project/project.mako

```diff
@@ -380,7 +414,8 @@ ${parent.javascript_bottom()}
         node: {
             hasChildren: ${ node['has_children'] | sjson, n },
             isRegistration: ${ node['is_registration'] | sjson, n },
-            tags: ${ node['tags'] | sjson, n }
+            tags: ${ node['tags'] | sjson, n },
+            alternativeCitations: ${ node['alternative_citations'] | sjson, n}
```

Comment:
As pointed out in https://github.com/CenterForOpenScience/osf.io/pull/4514/files#r47519048, prefer to fetch data through the API rather than using contextVars.